src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1575,6 +1575,7 @@ internal enum ErrorCode
         WRN_TupleBinopLiteralNameMismatch = 8383,
         ERR_TupleSizesMismatchForBinOps = 8384,
         ERR_ExprCannotBeFixed = 9385,
```

Comment:
>9385, [](start = 32, length = 5)

Not your PR but this looks wrong.

Reply:
I will file a bug. 

---
In reply to: [177497359](https://github.com/dotnet/roslyn/pull/25505#discussion_r177497359) [](ancestors = 177497359)

Reply:
[https://github.com/dotnet/roslyn/issues/25752](https://github.com/dotnet/roslyn/issues/25752)

---
In reply to: [177497404](https://github.com/dotnet/roslyn/pull/25505#discussion_r177497404) [](ancestors = 177497404,177497359)

Reply:
I'll go ahead and fix in this PR.

---
In reply to: [177497759](https://github.com/dotnet/roslyn/pull/25505#discussion_r177497759) [](ancestors = 177497759,177497404,177497359)